docs/laravel.md

```diff
@@ -71,8 +71,6 @@ The `octane:frankenphp` command can take the following options:
 * `--poll`: Use file system polling while watching in order to watch files over a network
 * `--log-level`: Log messages at or above the specified log level
 
-> Note: When using Laravel Octane environment variables will not be respected in your Caddyfile, instead Laravel Octane will add envrionment variables into the FrankenPHP process based on the above arguments. [See how here](https://github.com/laravel/octane/blob/2.x/src/Commands/StartFrankenPhpCommand.php#L92).
```

Comment:
I removed this part because this isn't true. Symfony Process passes system environment variables to the child process: https://symfony.com/doc/current/components/process.html#setting-environment-variables-for-processes

Reply:
Maybe should we add an example of how to customize the Caddyfile when using Octane?

Reply:
Yeah I think that would be helpful, but I think before I do that I might PR a change into Octane that changes some of the names of the Env variable binds in the stub caddyfile to be more clear.